Source/WebKit/UIProcess/RemoteLayerTree/mac/RemoteLayerTreeEventDispatcher.h

```diff
@@ -159,6 +161,9 @@ class RemoteLayerTreeEventDispatcher : public ThreadSafeRefCounted<RemoteLayerTr
     std::optional<DisplayLinkObserverID> m_displayRefreshObserverID;
     PAL::HysteresisActivity m_wheelEventActivityHysteresis;
 
+    std::atomic<bool> m_FingerDownIntervalIsActive = false;
```

Comment:
`m_fingerDownIntervalIsActive` etc (lowercase the letter after the underscore).

Reply:
Fixed thanks